pages/windows/stop-service.md

```diff
@@ -0,0 +1,17 @@
+# Stop-Service
+
+> Stops one or more running services.
+> This command can only be used through PowerShell.
+> More information: <https://learn.microsoft.com/powershell/module/microsoft.powershell.management/stop-service>.
+
+- Stop a service on the local computer:
+
+`Stop-Service -Name {{"sysmonlog"}}`
+
+- Stop a service by using the display name:
+
+`Stop-Service -DisplayName {{"telnet"}}`
```

Comment:
```suggestion
`Stop-Service -DisplayName "{{name}}"`
```

Reply:
Following the other suggestions would you rather user the following?

```suggestion
`Stop-Service -DisplayName "{{service_name}}"`
```